components/match2/wikis/rocketleague/legacy_match_list.lua

```diff
@@ -90,11 +90,11 @@ function LegacyMatchList.convertMatchMaps(frame)
 		end
 
 		if template ~= 'tbd' then
-			args['opponent' .. index] = MatchSubobjects.luaGetOpponent(frame, {
```

Comment:
#831 was merged today (legacy modules for R6). Could you take a look if any changes are required in this PR in those modu.les.

Reply:
None of those modules call custom opponent processing directly so no changes are needed